lib/ClangImporter/ImportEnumInfo.h

```diff
@@ -0,0 +1,101 @@
+//===--- EnumInfo.h - Information about importable Clang enums ------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file provides ImportEnumInfo, which describes a Clang enum ready to be
+// imported
+//
+//===----------------------------------------------------------------------===//
+#ifndef SWIFT_CLANG_IMPORT_ENUM_H
+#define SWIFT_CLANG_IMPORT_ENUM_H
+
+#include "swift/AST/Decl.h"
+#include "clang/AST/Attr.h"
+#include "clang/Basic/SourceLocation.h"
+#include "llvm/ADT/APSInt.h"
+#include "llvm/ADT/DenseMap.h"
+
+namespace clang {
+class EnumDecl;
+class Preprocessor;
+class MacroInfo;
+}
+
+namespace swift {
+namespace importer {
+
+/// \brief Describes how a particular C enumeration type will be imported
```

Comment:
Please don't use `\brief` in new files. Omitting it has exactly the same behavior.